services/web-app/components/mini-card/mini-card.module.scss

```
@@ -0,0 +1,45 @@
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/utilities/convert' as convert;
+@use '@carbon/react/scss/motion' as motion;
+
+@media screen and (prefers-reduced-motion: reduce) {
+  .card {
+    transition: none;
+  }
+}
+
+.card {
+  display: flex;
+  height: convert.rem(64px);
+  flex-direction: column;
+  justify-content: center;
+  background: theme.$layer-01;
+  transition: background motion.$duration-fast-02;
+}
+
+.card:hover {
+  background: theme.$background-hover;
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-right: spacing.$spacing-05;
+  padding-left: spacing.$spacing-05;
+}
+
+.title {
+  @include type.type-style('body-short-01');
+}
+
+.image {
```

```suggestion
.image image {
```
This should fix the image alignment being off